examples/simple_regional_private/main.tf

```diff
@@ -18,6 +18,11 @@ locals {
   cluster_type = "simple-regional-private"
 }
 
+provider "google" {
```

Comment:
This doesn't seem relevant to the PR as described -- what's going on here?

Reply:
This was added in order to support `location` field in `data "google_container_engine_versions" "zone"` and `data "google_container_engine_versions" "region"` in `main.tf`. Failing test was: https://concourse.infra.cft.tips/teams/cft/pipelines/terraform-google-kubernetes-engine/jobs/integration-tests/builds/450

HOWEVER: I will be reverting `location` back to `zone` and `region` and thus these changes as well. I will fix deprication warnings for `zone` and `region` in another PR sometime in the future.

Reply:
:+1: